pysc2/bin/replay_actions.py

```diff
@@ -216,16 +146,16 @@ def run(self):
                   self._print("Starting %s from player %s's perspective" % (
                       replay_name, player_id))
                   self.process_replay(controller, replay_data, map_data,
-                                      player_id)
+                                      player_id,info,replay_name)
```

Comment:
passing info and replay_name to process_replay to allow user to parse info data, replay_name for file naming